### PR TITLE
add an example of using cleanedRequest in getAccessToken

### PR DESCRIPTION
**Pull request by buildmaster**
Adding example of using cleanedRequest as created in PR zapier/zapier#16747

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,451 @@
+{
```
Comment (bryanhelmig) [inline package-lock.json]: Are we doing `package-lock.json`?

**Review by bryanhelmig — APPROVED** [review]

File: package-lock.json (hunk 1 above)
Comment (buildmaster) [reply package-lock.json]: sorry added by mistake. but do we want to add package-lock? @bcooksey?

Comment (bryanhelmig) [reply package-lock.json]: Probably nuke from this PR and see if there is a macro change we want to make here? Could be an issue on the core / cli repo.

**Review by bryanhelmig — APPROVED** [review]

File: package-lock.json (hunk 1 above)
Comment (bcooksey) [reply package-lock.json]: For example apps I'd say no, just so we keep the amount of required files to a minimum and devs don't assume the platform needs a package-lock.json to function.

**Pull request merged**